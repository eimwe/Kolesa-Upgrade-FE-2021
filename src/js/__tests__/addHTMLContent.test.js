const divNode = document.createElement('DIV');

divNode.id = 'app';
const getDivNode = () => document.body.appendChild(divNode);
const clearDivNode = () => {
    divNode.remove();

    return true;
};

describe('Проверка блока #app', () => {
    beforeAll(() => {
        getDivNode();
    });

    test('Формат принимаемых данных - строка', () => {
        const data = 'mock';

        expect(typeof data).toBe('string');
    });

    test('Вставка контента в #app', () => {
        const data = 'mock';

        divNode.innerHTML = data;

        expect(divNode).toMatchSnapshot();
    });

    afterAll(() => {
        clearDivNode();
    });
});
