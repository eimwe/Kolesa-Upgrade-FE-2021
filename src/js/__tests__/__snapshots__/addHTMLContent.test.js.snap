// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверка блока #app Вставка контента в #app 1`] = `
<div
  id="app"
>
  mock
</div>
`;
